ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: cli-ocm
     namespace: ci
     tag: latest
+  cli-operator-sdk:
+    name: cli-operator-sdk
+    namespace: ocp
+    tag: v1.30.0
```

Comment:
Note: version 1.30.0 mismatches with what we actual have in FIO, which is 1.27.0. However, when we specify 1.27.0 as the tag, CI fails to find the image.

Trying a new image to see if that gets us further.

Reply:
We could also try the following instead:

```yaml
base_images:
  operator-sdk:
    name: "4.18"
    namespace: origin
    tag: operator-sdk
```